docs/tutorial/bit_extraction.md

```diff
@@ -0,0 +1,122 @@
+# Bit Extraction
+
+Some applications require directly manipulating bits of integers. Concrete provides bit extraction operation for such applications.
```

Comment:
Could be a more a bit more descriptive, since the feature is not described in text after. E.g.

Bit extraction is capable of extracting a slice of bits from an integer. The lowest significant bit has index 0. The  slice highest significant extracted bit index is always required. The slice step parameter is supported.
The cost of this operation is proportional to the highest significant bit index.

Reply:
> highest significant extracted bit index is always required

This is not true, you can do `fhe.bits(x)[3:]`. It's not required for unsigned.

Reply:
> The slice step parameter is supported

We already have an example of this in the bottom, no need to put it here.

Reply:
I'll add this paragraph:
```
Bit extraction is capable of extracting a slice of bits from an integer. Index 0 corresponds to the lowest significant bit. The cost of this operation is proportional to the highest significant bit index.
```